src/main/kotlin/pl/edu/agh/gem/external/dto/ValidationMessage.kt

```diff
@@ -0,0 +1,15 @@
+package pl.edu.agh.gem.external.dto
+
+class ValidationMessage private constructor() {
```

Comment:
we could you object instead of class with compaion object

Reply:
will add in last pr